hazelcast/src/main/java/com/hazelcast/collection/impl/queue/QueueContainer.java

```diff
@@ -1250,4 +1249,14 @@ public String toString() {
                 + ", lastIdLoaded=" + lastIdLoaded
                 + '}';
     }
+
+    /**
+     * @since 5.0.1
```

Comment:
Do we have to backport this to 5.0.1? While designing the protocol, we said that we won't do changes for the patch versions unless we kinda have to. I know there were some exceptions to this rule, but I would like to stick to it here.

Reply:
Thanks, I didn't know about such policy. In that case we won't backport this change to 5.0 maintenance (cc. @emre-aydin ).